ui/app/templates/components/clients/running-total.hbs

```diff
@@ -36,68 +51,15 @@
         </p>
       </div>
     </div>
-
-    <div class="single-chart-grid">
-      <div class={{concat (unless this.hasAverageNewClients "chart-empty-state ") "chart-container-wide"}}>
-        <Clients::VerticalBarChart
-          @dataset={{if this.hasAverageNewClients this.byMonthNewClients false}}
-          @chartLegend={{this.runningTotalLegend}}
-          @noDataTitle="No new clients"
-          @noDataMessage={{concat
-            "There is no new client data available for this "
-            (if @selectedAuthMethod "auth method" "namespace")
-            " in this date range"
-          }}
-        />
-      </div>
-
-      <div class="chart-subTitle">
-        <h2 class="chart-title">New monthly clients</h2>
-        <p class="chart-subtext">
-          Clients which interacted with Vault for the first time during this date range, displayed per month.
-        </p>
-      </div>
-
-      {{#if this.hasAverageNewClients}}
-        <div class="data-details-top" data-test-running-new-entity>
-          <h3 class="data-details">Average new entity clients per month</h3>
-          <p class="data-details">
-            {{format-number this.entityClientData.averageNewClients}}
-          </p>
-        </div>
-
-        <div class="data-details-bottom" data-test-running-new-nonentity>
-          <h3 class="data-details">Average new non-entity clients per month</h3>
-          <p class="data-details">
-            {{format-number this.nonEntityClientData.averageNewClients}}
-          </p>
-        </div>
-      {{/if}}
-
-      <div class="timestamp" data-test-running-total-timestamp>
-        {{#if @responseTimestamp}}
-          Updated
-          {{date-format @responseTimestamp "MMM d yyyy, h:mm:ss aaa" withTimeZone=true}}
-        {{/if}}
-      </div>
```

Comment:
Each chart should have a timestamp, so this should remain

Reply:
Thanks I added it back to the component!